### ISSUE TITLE
iSEE app for snRNA-seq data

### ISSUE THREAD

**Issue by lcolladotor** [LieberInstitute/Habenula_Pilot#12]
- [ ] Create a subsetted SCE object similar to https://github.com/LieberInstitute/septum_lateral/blob/main/snRNAseq_mouse/code/03_iSEE_apps/snRNA-seq_app/subset.R which is based off https://github.com/LieberInstitute/septum_lateral/blob/b0893a51cf71c685a66aa3d4165d2a829badabd7/snRNAseq_mouse/code/02_analyses/03_reducedDimensions_clustering.R#L998-L1070. Aka, keep only `logcounts` and drop any large memory components. The final object should ideally be less than 2.5 GB of RAM.
- [ ] Adapt the `app.R` script from https://github.com/LieberInstitute/septum_lateral/tree/main/snRNAseq_mouse/code/03_iSEE_apps/snRNA-seq_app, then test it locally.
- [ ] From the local app, find the display settings that we'd like to share by default (talk with @heenadivecha about this). You'll need a list of genes to highlight on the default heatmap (max 10 genes). 
- [ ] Then adapt the `deploy.R` script from https://github.com/LieberInstitute/septum_lateral/tree/main/snRNAseq_mouse/code/03_iSEE_apps/snRNA-seq_app.
- [ ] Edit the shinyapps.io settings to match those we typically use for `iSEE` apps (talk to @heenadivecha again).

**Comment by lcolladotor** [comment]
I think that we can close this issue, right Louise @lahuuki ?

**Comment by lahuuki** [comment]
Yes, iSEE app is functional h[ttps://libd.shinyapps.io/habenulaPilot_snRNAseq/](https://libd.shinyapps.io/habenulaPilot_snRNAseq/)
